dist/fluxo.js

```diff
@@ -358,6 +361,16 @@ Fluxo.CollectionStore = Fluxo.Base.extend(
 
     if (storeID) {
       foundStore = this.findWhere({ id: storeID });
+
+      if (!foundStore) {
+        this.stores.some(function(store) {
+          if (store.cid === storeID) {
+            foundStore = store;
+
+            return true;
+          }
+        });
+      }
```

Comment:
I'm not really happy with this but there is no `Array#findFirst` on ECMAScript, we need to emulate it.